superset/models/dashboard.py

```diff
@@ -361,15 +364,16 @@ def export_dashboards(  # pylint: disable=too-many-locals
     @classmethod
     def get(cls, id_or_slug: str) -> Dashboard:
         session = db.session()
-        qry = session.query(Dashboard)
-        if id_or_slug.isdigit():
-            qry = qry.filter_by(id=int(id_or_slug))
-        else:
-            qry = qry.filter_by(slug=id_or_slug)
-
+        qry = session.query(Dashboard).filter(id_or_slug_filter(id_or_slug))
         return qry.one_or_none()
 
 
+def id_or_slug_filter(id_or_slug: str) -> BinaryExpression:
```

Comment:
nit: `def id_or_slug_filter(id_or_slug: Union[int, str]) -> BinaryExpression:`

Reply:
It actually will only accept a string